src/databricks/labs/ucx/install.py

```diff
@@ -907,6 +907,75 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def uninstall(self):
+        if not self._prompts or self._question(
+            "Do you want to uninstall ucx from the workspace too, this would "
+            "remove ucx project folder, dashboards, queries and jobs",
+            default="yes",
+        ):
+            logger.info(f"UnInstalling UCX from workspace v{self._version}")
+            try:
+                self._ws.workspace.get_status(self.config_file)
+                self._ws.workspace.get_status(self._install_folder)
+                self._ws.workspace.get_status(self._state._state_file)
+            except NotFound:
+                logger.error(
+                    f"Check if {self._install_folder} is present along with {self.config_file} and "
+                    f"{self._state._state_file}."
+                )
+                return
+            self._remove_database()
+            self._remove_jobs()
+            self._remove_warehouse()
+            self._remove_install_folder()
+            logger.info("UnInstalling UCX complete")
+
+    def _remove_database(self):
+        from databricks.labs import ucx
+
+        if (
+            not self._prompts
+            or self._question(
+                f"Do you want to delete the inventory database {self._current_config.inventory_database} too?",
+                default="yes",
+            )
+            == "yes"
+        ):
+            logger.info(f"Deleting inventory database {self._current_config.inventory_database}")
+            if self._sql_backend is None:
+                self._sql_backend = StatementExecutionBackend(self._ws, self._current_config.warehouse_id)
+            deployer = SchemaDeployer(self._sql_backend, self._current_config.inventory_database, ucx)
+            deployer.delete_schema()
+
+    def _remove_jobs(self):
+        logger.info("Deleting jobs")
+        if not self._state.jobs:
+            logger.error("No jobs present or jobs already deleted")
+            return
+        for step_name, job_id in self._state.jobs.items():
+            try:
+                logger.info(f"Deleting {step_name} job_id={job_id}.")
+                self._ws.jobs.delete(job_id)
+            except InvalidParameterValue:
+                logger.error(f"Error deleting job {step_name} job_id={job_id}.")
+                continue
+
+    def _remove_warehouse(self):
+        try:
+            warehouse_name = self._ws.warehouses.get(id=self._current_config.warehouse_id).name
+            if warehouse_name.startswith("Unity Catalog Migration"):
```

Comment:
can you extract this prefix into a constant, otherwise developers will forget to change the name here as well